test/integration/ComputeFlow.test.ts

```diff
@@ -16,6 +16,7 @@ import { Account, DDO, Metadata } from '../../src/lib'
 import { Cluster, Container, Server } from '../../src/ocean/Compute'
 import { LoggerInstance } from '../../src/utils'
 import { ComputeInput } from '../../src/ocean/interfaces/ComputeInput'
+import fetch from 'node-fetch'
```

Comment:
I noticed that you've added in `node-fetch` here which we have been removing in the other pull request. It might be better to use `cross-fetch` here instead?